javascript/ql/test/library-tests/Arrays/DataFlow.expected

```diff
@@ -1,11 +1,11 @@
 legacyDataFlowDifference
+| arrays.js:2:16:2:23 | "source" | arrays.js:39:8:39:24 | arr4_spread.pop() | only flow with OLD data flow library |
```

Comment:
Are you planning to add this step to the new library?

Reply:
I'm planning to fix this missing flow by adding general handling of the spread operator, so individual models don't have to specify how to handle spread operators. For example if a model specifies `Argument[1]` somewhere, and it applies to call that has a spread argument at position 0 or 1, then it should Just Work. The model shouldn't have to specify something like `Argument[spread]` which is what the legacy model currently does.